docs/spec-files/requirements.md

```diff
@@ -2,7 +2,7 @@
 
 The [DIDComm Messaging](.) design attempts to be:
 
-1. **Secure** (tamper-proof; uses best-of-breed crypto; allows parties to talk both on and off the record.)
+1. **Secure** (specifically, MUST have message confidentiality,message integrity, and message authentication; MUST use best-of-breed crypto; MUST allows parties to emit both repudiable and non-repudiable messages.)
```

Comment:
It would be fair to say about current state of forward secrecy as it often assumed in *secure* protocols.

Reply:
There is already a separate PR about PFS; see #307. It is in the Implementer's Guide rather than the spec, because PFS is not a requirement. But I agree it is relevant here. I have just added an additional update to the text here to mention it briefly; if/when the PFS PR is merged, we can update this with a better link.